Synergism.css

```diff
@@ -3714,3 +3714,14 @@ img#singularityPerksIcon {
     max-width: 500%;
     animation: rotation 2s infinite linear;
 }
+
+.hoveredBlueberryLoadout1 div.bbPurchasedLoadout1,
```

Comment:
it's probably possible to simplify this entire block of css to
```css
div[class^="hoveredBlueberryLoadout"] div[class^="bbPurchasedLoadout"] {
```

Reply:
It'll match the indices if we do that?  If it matches `hoveredBlueberryLoadout1` with `bbPurchasedLoadout2`, the highlighting effect between loadouts will get mixed.

Reply:
Further experiment:  it's actually not matching _at all_ for the highlighting, even if I drop one of the two.  Maybe it's because neither is first in the class list?